src/include/sof/ipc.h

```diff
@@ -96,7 +96,7 @@ struct dai_config;
 			trace_ipc("ipc: hdr 0x%x rx (%d) > tx (%d)",	\
 				  rx->cmd, rx_size, tx->size);		\
 		} else if (tx->size > rx_size) {			\
-			memcpy(rx, tx, tx->size);			\
+			memcpy(rx, tx, rx_size);			\
```

Comment:
@keyonjie should this be in a separate patch?

Reply:
It's ok, I'll merge in one PR.